posts/postsRouter.js

```diff
@@ -0,0 +1,102 @@
+const express = require("express");
+
+const Posts = require("./postsModel"); //accessing model. add in postModel file
+
+const router = express.Router();
+
+router.get("/", (req, res) => {
+  Posts.find(req.query)
+    .then(hubs => {
```

Comment:
`hubs` is just a variable that we can call anything. However, try to keep your variable names accurate/descriptive. For example, the db operations brings back 'data' or better yet 'posts'.